simplepypi/bin/simplepypi

```diff
@@ -105,10 +105,13 @@ class PypiPackageHandler(RequestHandler):
             return
 
         if os.path.exists(package_path):
-            self.write(
-                '<html><body>{}</body></html>'.format(
-                    generate_package_link(package, versions[-1])
-                ))
+            self.write('<html><body>')
+            for version in versions:
+                self.write(
+                    '{}'.format(
+                        generate_package_link(package, version)
+                    ))
```

Comment:
I guess there is no need for this formatting, given we're dealing only with output of the `generate_package_link` function?
```
self.write(generate_package_link(package, version))
```

Reply:
Good call, that was dumb.